dev-python/qdarkstyle/qdarkstyle-2.7.ebuild

```diff
@@ -0,0 +1,22 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{5,6,7,8} )
```

Comment:
Skip python3.5 as it's gonna get removed soon anyway :)

Reply:
If I do that I would also have to remove py3_5 from spyder and spyder-kernels because this is a dependency of those packages. And I seem to be getting conflicting advice on this: First @flappyports told me to drop py3_5, then when I did, @hartwork asked me to not drop py3_5 on existing packages (those packages that only needed a version bump). So I re-added py3_5. 

I'm sorry but I cannot do both (repoman will yell at me if I do). I cannot keep py3_5 on existing packages (spyder, spyder-kernels) and at the same time not add it to the new dependencies I added (qdarkstyle, wurlitzer etc).

Reply:
I wasn't aware this was part of some bigger update cluster. I see history in spyder PR, I should probably comment there as well, but what if you dropped python3.5 from the NEW version of spyder? I dont see value in adding 20 python packages with python3.5 just so it can be removed from them in a month.

Nothing seems to be depending on spyder so dropping py3.5 for new version shouldnt break anything. Just my thought.

Reply:
Why I suggested keeping Python 3.5 unless broken: https://github.com/gentoo/gentoo/pull/14014#discussion_r358383748

Reply:
Also, I thought python3_5 does not reach end-of-life unil September next year?
Or is it going to be removed simultaneous with python2_7?

Reply:
That sounds aright with regard to Python upstream: https://devguide.python.org/#status-of-python-branches

Reply:
@gentoo/python has announced their desire and intention to drop python3.5 from Gentoo. It doesn't matter what upstream does, if our Python team isn't supporting it anymore. @flappyports is part of the python team btw so I'd listen to him... not trying to overrule anyone, but to explain where these intentions are coming from. Yes, it is possible to drop py3.5 from the tree simultaneously, but if our python team has asked you to start casually dropping py3.5 where possible and definitely not introduce it on new packages, you should honor their request and listen to it - it's because of a reason.

I personally don't have a strong opinion on this, just trying to work together with everyone :)

Reply:
> I personally don't have a strong opinion on this, just trying to work together with everyone :)

:+1: Same for me regarding Python 3.5.

> if our python team has asked you to start casually dropping py3.5 where possible and definitely not introduce it on new packages, you should honor their request and listen to it

Do you have a link/source for this?

Reply:
Alright, py3_5 is gone

Reply:
> > if our python team has asked you to start casually dropping py3.5 where possible and definitely not introduce it on new packages, you should honor their request and listen to it
> 
> Do you have a link/source for this?

Apart from flappyport's comment in the other PR not really. There's been talk in either #gentoo-dev or #gentoo-python if you wanna check IRC logs...

Reply:
Okay thanks!